svgparser.py

```diff
@@ -1742,7 +1780,7 @@ class SVGLoader(SVGGeometryContainer):
 
     # TODO: Fix so that this is done like in the original plugin (e.g. do_colormanage)
     # def __init__(self, blender_context, svg_filepath, origin="TL", depth="1.94397pt"):
-    def __init__(self, blender_context, svg_filepath, origin="TL", depth="0"):
+    def __init__(self, blender_context, svg_filepath, origin="TL", depth="0", scale=0):
```

Comment:
Note:  I also wanted to be able to mess with the scale conversion factor more easily, so I have made that a parameter here.